docs/event_sequence.md

```diff
@@ -16,12 +16,27 @@ further:
 ## Test error short-circuiting
 
 A phase raising an exception won't kill the test, but will initiate a
-short-circuit.
+terminal short-circuit. Phases are additionally terminal if they return
+`htf.PhaseResult.STOP` or if they exceed their Phase timeout.
 
-When a phase raises an exception or returns `htf.PhaseResult.STOP`:
-* If the phase was `test_start`, then we skip to plug `tearDown`.
-* If the phase wasn't `test_start`, then we skip to the teardown phase.
-* Test outcome is ERROR for output callbacks.
+If a `test_start` phase is terminal, then we skip the plug `tearDown`.
```

Comment:
Is this right or do you mean we skip the teardown phases and skip to plug teardown? Seems like the test start plugs would still be torn down.

Reply:
You are correct; fixing the wording.